packages/zoltan2/sphynx/src/Zoltan2_Sphynx.hpp

```diff
@@ -558,14 +563,41 @@ namespace Zoltan2 {
     anasaziParams.set("Orthogonalization", ortho);
     anasaziParams.set("Use Locking", lock);
     anasaziParams.set("Relative Locking Tolerance", relLockTol);
+    anasaziParams.set("Full Ortho", useFullOrtho);
 
+    typedef typename Adapter::user_t graph_type;
+    typedef typename graph_type::global_ordinal_type GO;
```

Comment:
Global ordinal is already defined as gno_t in line 105.  Do we need GO?

Reply:
Resolved, thanks!

Reply:
No, we don't :+1: